### PR TITLE
Launcher version in production

### PR DESCRIPTION
**Pull request by GrantPerkins**

### CODE REVIEW

File: packages/launcher/package.json [hunk 1]
```diff
@@ -77,7 +78,7 @@
     "postinstall": "electron-builder install-app-deps",
     "electron:dev": "concurrently \"PORT=2000 BROWSER=none yarn start\" \"wait-on http://localhost:2000 && tsc -p electron -w\" \"wait-on http://localhost:2000 && tsc -p electron && electron .\"",
     "electron:dev:win": "yarn electron:copy-icon && concurrently \"set PORT=2000 BROWSER=none && yarn start\" \"wait-on http://localhost:2000 && tsc -p electron -w\" \"wait-on http://localhost:2000 && tsc -p electron && electron .\"",
-    "electron:build": "yarn build && tsc -p electron && electron-builder",
+    "electron:build": "yarn build && tsc -p electron && ./node_modules/.bin/electron-builder --projectDir .",
```
Comment (AustinShalit) [inline packages/launcher/package.json:80]: What does this change do?

File: packages/launcher/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "launcher",
   "productName": "Axon",
   "version": "0.0.1",
+  "buildVersion": "0.0.1",
```
Comment (AustinShalit) [inline packages/launcher/package.json]: Do we need to specify twice? Seems like some folks have used `process.env.npm_package_version` with success? Can we try that?

Comment (GrantPerkins) [reply packages/launcher/package.json]: no, this doesn't do anything. I tried the environment variable, but it does not exist in production. app.getVersion() works.

File: packages/launcher/package.json (hunk 1 above)
Comment (GrantPerkins) [reply packages/launcher/package.json:80]: "If you don’t have dist npm script in your package.json, call ./node_modules/.bin/electron-builder directly."

https://www.electron.build/multi-platform-build

Allows for the app version to actually be recognized by the electron-builder.

File: packages/launcher/electron/main.ts [hunk 3]
```diff
@@ -102,5 +102,5 @@ ipcMain.on("request-tags", (event) => {
 
 ipcMain.on("launcher-version", (event) => {
   // gets build version for launcher. ENV variables are difficult with electron.
-  event.returnValue = process.env.npm_package_version;
+  event.returnValue = app.getVersion();
```
Comment (GrantPerkins) [inline packages/launcher/electron/main.ts:105]: @AustinShalit see the change made here. This env does not work in production. New solution is cleaner.

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**